Source/ASPagerNode.mm

```diff
@@ -29,6 +29,7 @@ @interface ASPagerNode () <ASCollectionDataSource, ASCollectionDelegate, ASColle
     unsigned nodeBlockAtIndex:1;
     unsigned nodeAtIndex:1;
   } _pagerDataSourceFlags;
+    BOOL _allowsAutomaticInsetsAdjustment;
```

Comment:
Should we broaden the flags struct and add this into it?

Reply:
I didn't want to conflate the data source with the object BOOL properties.